Project1/preview-cards/index.html

```diff
@@ -15,18 +19,34 @@
   </style>
 </head>
 <body>
-  
-  Sedans
-  Choose a sedan for its affordability and excellent fuel economy. Ideal for cruising in the city 
-  or on your next road trip.
-
-  SUVs
-  Take an SUV for its spacious interior, power, and versatility. Perfect for your next family vacation 
-  and off-road adventures.
-
-  Luxury
-  Cruise in the best car brands without the bloated prices. Enjoy the enhanced comfort of a luxury 
-  rental and arrive in style.
-  
+  <div class="row">
+    <div class="sedan">
+      <img src="./images/icon-sedans.svg">
+      <h1>Sedans</h1>
+      <p>
+      Choose a sedan for its affordability and excellent fuel economy. Ideal for cruising in the city 
+      or on your next road trip.
+      </p>
+      <p id="button"  class="buttonsedan">Learn More</p>
```

Comment:
pattern na klasy w CSS to sneak-case, poprawi to clean code w twoim projekcie
```suggestion
      <p id="button"  class="button-sedan">Learn More</p>
```